### ISSUE TITLE
question of life or death FOR me

### ISSUE THREAD

**Issue by esmanning** [carmls/snacs-guidelines#90]
> It was a question of life or death FOR me: I had scarcely enough drinking water to last a week.
Little Prince chapter 2
sent_id = lpp_1943.40

Ambiguous whether the PP attaches to "question" or "life or death", though the former reading seems more likely. Torn between Experiencer, Beneficiary, or Experiencer~Beneficiary.

**Comment by nschneid** [comment]
I would paraphrase as "FOR me it was a question of life or death".

Borders on the fuzziness of FOR noted in #70

**Comment by nschneid** [comment]
In the context, is the character in the story pondering a question, or is this just saying something in retrospect was a matter of life or death?

**Comment by aryamanarora** [comment]
I thought it fell under the "from the perspective of" use of *for* (267 in the guidelines), when we talked about it in adjudication. Fun fact, I have this exact same issue in annotating the Hindi version.

**Comment by nschneid** [comment]
Yeah, looking at the context I think it's fair to interpret it as "I was facing a life or death situation", which fits Experiencer~>Beneficiary.

**Comment by caffeine96** [comment]
Encountered the same issue in Gujarati as well. I had to choose between Beneficiary and Experiencer~>Beneficiary. I finally went with Beneficiary. My argument being that "a question of life and death for XYZ" implies a disadvantageous situation for XYZ.
